### PR TITLE
Updating config to use prettier and updated dependencies

### PR DESCRIPTION
**Pull request by mmellado**

### CODE REVIEW

File: eslintrc.json [hunk 1]
```diff
@@ -37,19 +39,10 @@
 
   },
   "rules": {
-    "array-bracket-spacing": [2],
```
Comment (elvisvoer) [inline eslintrc.json]: Is there a way you could explain why some rules have been removed while some stayed?

Comment (mmellado) [reply eslintrc.json]: All styling rules were removed as they are handled by prettier, The rest of the rules, which are more about imports, validation, proper practices etc are kept, as these are not handled by prettier

**Comment by mmellado** [comment]
I messed something mup and I have no patience to fix it. closing and creating a new PR....

**Comment by alexjeffburke** [comment]
@mmellado ..ok. Any chance you could mention what you found though? Would be quite useful for those of us following along but without the time to look into this in depth.

**Comment by mmellado** [comment]
I messed up merges and ended up in a conficting branch that was unabel to be merged and causing merge conflicts... I'll regrenerate the PR in a bit explaining eveyrthing

**Pull request closed without merging**